homeassistant/components/thermostat/zwave.py

```diff
@@ -0,0 +1,108 @@
+"""
+ZWave Thermostat.
+
+"""
+
+# Because we do not compile openzwave on CI
+# pylint: disable=import-error
+from homeassistant.components.thermostat import DOMAIN
+from homeassistant.components.thermostat import ThermostatDevice
+from homeassistant.components.zwave import (
+    ATTR_NODE_ID, ATTR_VALUE_ID, NETWORK,
+    ZWaveDeviceEntity)
+from homeassistant.const import TEMP_FAHRENHEIT
+
+CONF_NAME = 'name'
+DEFAULT_NAME = 'ZWave Thermostat'
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the ZWave thermostats."""
+    if discovery_info is None or NETWORK is None:
+        return
+
+    node = NETWORK.nodes[discovery_info[ATTR_NODE_ID]]
+    value = node.values[discovery_info[ATTR_VALUE_ID]]
+    value.set_change_verified(False)
+    add_devices([ZWaveThermostat(value)])
+
+
+# pylint: disable=too-many-arguments
+class ZWaveThermostat(ZWaveDeviceEntity, ThermostatDevice):
+    """Represents a HeatControl thermostat."""
+    def __init__(self, value):
+        from openzwave.network import ZWaveNetwork
+        from pydispatch import dispatcher
+        from homeassistant.helpers.temperature import convert
```

Comment:
I'm just talking about the `homeassistant` import. It got added here when you accepted the PR by @luxus